scss/footer.scss

```diff
@@ -0,0 +1,31 @@
+.footer {
+  background: $blue;
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  height: 200px;
+  justify-content: center;
+  position: relative;
+  width: 100%;
+}
+
+.footer-text {
+  align-self: center;
+  color: white;
+  font-size: 24px;
+  margin-bottom: 30px;
+  position: relative;
```

Comment:
not use a position relative unless you are going to attatch an absolute to the item